src/source.h

```diff
@@ -0,0 +1,74 @@
+/// \brief The tally class and helper functions.
+///
+/// The tally class is in essesence a structure containing attributes
+/// related to tallies.
+
+#ifndef PYNE_IQ4M73STINHJDPRV6KWUZZXOYE
+#define PYNE_IQ4M73STINHJDPRV6KWUZZXOYE
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <set>
+#include <string>
+
+#ifndef PYNE_IS_AMALGAMATED
+#include "h5wrap.h"
+#include "utils.h"
+#endif
+
+namespace pyne {
+
+class Source {
+ public:
+  /// Source Constructors
+  Source();  /// empty constructor
+
+  ~Source();  /// default destructor
+
+  // mcnp tally
+  virtual std::string mcnp() {}
+
+  // fluka tally
+  virtual std::string fluka() {}
+};
+
+class PointSource : public Source {
+ public:
+  /// Point Source Constructors
+  PointSource(double x = 0, double y = 0, double z = 0, double i = 0,
+              double j = 0, double k = 0, double E = 14, std::string p = "n",
```

Comment:
I think the MCNP default is actually 14.1 MeV isn't it???  Do we want to allow defaults?

Reply:
![Screenshot 2020-04-13 10 13 08](https://user-images.githubusercontent.com/15145274/79132270-a3225700-7d6f-11ea-91d9-fe2901892311.png)
User manual says 14MeV.


I don't know about all defaults, but some would be nice, such as the isotropy of the source...